website/docs/using-unleash/compliance/fedramp.mdx

```diff
@@ -0,0 +1,54 @@
+|----------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [AU-02 Event Logging](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=AU-2)           | Unleash provides detailed [audit logs and event tracking](/reference/events), accessible through the Admin UI or exportable for integration with other systems.                                      |
+| [AU-12 Audit Record Generation](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=AU-12) | Unleash provides detailed [audit logs and event tracking](/reference/events) to support auditors with the required evidence. Depending on the audit automation level, the evidence can be collected using the Unleash Admin UI, or exported to other systems. |
+
+## Security Assessment and Authorization
+
+| **FedRAMP Control**                                                                             | **Unleash Features**                                                                                                                       |
+|-------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [CA-8 Penetration Testing](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=CA-8) | Unleash conducts annual penetration testing by external auditors. Results are available for download from the Trust Center.                                                     |
+
+## Configuration Management
+
+| **FedRAMP Control**                                                                                          | **Unleash Features**                                                                                                                                          |
+|--------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [CM-02 Baseline Configuration](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=CM-2)         | Unleash provides [Export](/how-to/how-to-environment-import-export) functionality that facilitates keeping a configuration snapshot in the audit records.                                            |
```

Comment:
@dsusa72 is this the export functionality we should link to? Do we want to distinguish between system configuration vs. feature flag configuration? For system config, we could also mention the Terraform provider?

Reply:
Melinda and I met w/Chris for inputs. TL;DR it's worth mentioning Export feature and Terraform, but not Helm charts.